frontend/src/entity-models/const-resources/validation-constraints.ts

```diff
@@ -2,5 +2,5 @@ export const ValidationConstants = {
     minLengthName: 4,
     minLengthPassword: 8,
     minLengthEmail: 8,
-    emailRegex: '^[a-z0-9._%+-]+@[a-z0-9.-]+\\.[a-z]{2,4}$'
+    emailRegex: '^[a-zA-Z0-9._%+-]+@[a-z0-9.-]+\\.[a-z]{2,4}$'
```

Comment:
This regex is invalid:
- `\\` after domain?
- i suggest using `^[\w-\.]+@([\w-]+\.)+[\w-]{2,3}$` instead

Reply:
Why this regex is invalid? You can't use "\\" symbols in domain. Just try it

Reply:
nvm, i'm stupid